### ISSUE TITLE
C77 on 0.68.x test failure

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#5954]
## Description 
When running browser tests, there are two failures

## Steps to Reproduce

   1. Build https://github.com/brave/brave-browser/pull/5943
   2. Run `npm run test brave_browser_tests -- Release`
   3. See two failures

## Actual result:
```
2 tests failed:
    BraveClearDataOnExitTwoBrowsersTest.OneGuest (../../brave/browser/browsing_data/brave_clear_browsing_data_browsertest.cc:335)
    BraveClearDataOnExitTwoBrowsersTest.OneGuestExitsLast (../../brave/browser/browsing_data/brave_clear_browsing_data_browsertest.cc:352)
```

## Expected result:
No failures

## Reproduces how often: 
100%

**Comment by petemill** [comment]
As @mkarolin found, pulling in https://github.com/brave/brave-core/pull/2957 fixes this, which was done on 0.69.x with c77 and **fixed exactly the same test issues**. What 2957 does is removes calls to SetWebUIProperty.

However, 0.68.x NTP is too far away from 0.70.x to easily merge 2957 (which is on 0.70).

#### 0.69 (can easily get 2957 cherry-picked to)

[NTP: Added preferences and toggles for Clock / Stats / TopSites](https://github.com/brave/brave-core/pull/2762)
[NTP: Move from brave-ui to brave-core](https://github.com/brave/brave-core/pull/2857)
[NTP: Make menu keyboard accessible](https://github.com/brave/brave-core/pull/2814)

#### 0.68

[NTP: Added preference toggle for background image](https://github.com/brave/brave-core/pull/2668)


### Our options are

1. disable those tests  in 68 and hope we don't hit the error in real usage
    - Pro: extremely quick
    - Con: Not great to disable tests, and we still don't know why it's an issue. However, it does seem to only be affecting Guest (and possibly Tor) windows.
2. pull in 2957 to 68 and whatever else it needs (most of the 69 list)
     - Pro: fixes the issue
     - Con: bring in more untested features, although a known set affecting the NTP
3. Create a new version of 2957 against 68 that uses current 68 code
i.e. migrate 0.68 to _not_ use SetWebUIProperty
      - Pro: only changes backend code, not bringing in new features
      - Con: May take a few hours to complete

**Comment by bsclifton** [comment]
Fixed with https://github.com/brave/brave-browser/issues/5954
